posthog/tasks/usage_report.py

```diff
@@ -612,6 +616,28 @@ def get_teams_with_rows_synced_in_period(begin: datetime, end: datetime) -> list
     )
 
 
+@timed_log()
+@retry(tries=QUERY_RETRIES, delay=QUERY_RETRY_DELAY, backoff=QUERY_RETRY_BACKOFF)
+def get_teams_with_posthog_exceptions_captured_in_period(
+    begin: datetime,
+    end: datetime,
+) -> list[tuple[int, int]]:
+    results = sync_execute(
+        """
+        SELECT team_id, COUNT() as count
+        FROM events
+        WHERE event = '$exception' AND timestamp between %(begin)s AND %(end)s
+        AND not(JSONHas(properties, '$sentry_event_id'))
```

Comment:
can i get exceptions for free by setting $sentry_event_id?

Reply:
Chatted about this in person but just to close the loop. This just adds a new aggregation to the report. How we decide to bill based on those aggregations is different.

This is also limited to `event = '$exception'` which we don't yet charge for